### PR TITLE
Update gradle and other dependencies for CameraX samples.

### PR DESCRIPTION
**Pull request by donovanfm**
All changes are from running Upgrade Assistant suggestions, with the exception of updating TensorFlow Lite dependencies in CameraXAdvanced which was manual.

### CODE REVIEW

**Review by madebymozart — APPROVED** [review]
LGTM!

File: CameraXBasic/build.gradle [hunk 1]
```diff
@@ -26,9 +26,9 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.2.2'
+        classpath 'com.android.tools.build:gradle:7.2.2'
```
Comment (Stan4402) [inline CameraXBasic/build.gradle:29]: com.android.tools.build:gradle:7.2.2'

File: CameraXAdvanced/tflite/build.gradle [hunk 2]
```diff
@@ -66,9 +65,9 @@ dependencies {
 
 
     // Tensorflow lite dependencies
-    implementation 'org.tensorflow:tensorflow-lite:0.0.0-nightly-SNAPSHOT'
-    implementation 'org.tensorflow:tensorflow-lite-gpu:0.0.0-nightly-SNAPSHOT'
-    implementation 'org.tensorflow:tensorflow-lite-support:0.0.0-nightly-SNAPSHOT'
+    implementation 'org.tensorflow:tensorflow-lite:2.9.0'
+    implementation 'org.tensorflow:tensorflow-lite-gpu:2.9.0'
+    implementation 'org.tensorflow:tensorflow-lite-support:0.4.2'
```
Comment (Stan4402) [inline CameraXAdvanced/tflite/build.gradle:70]: 'org.tensorflow:tensorflow-lite-support:0.4.2'

File: CameraXAdvanced/tflite/build.gradle [hunk 3]
```diff
@@ -66,9 +65,9 @@ dependencies {
 
 
     // Tensorflow lite dependencies
-    implementation 'org.tensorflow:tensorflow-lite:0.0.0-nightly-SNAPSHOT'
-    implementation 'org.tensorflow:tensorflow-lite-gpu:0.0.0-nightly-SNAPSHOT'
-    implementation 'org.tensorflow:tensorflow-lite-support:0.0.0-nightly-SNAPSHOT'
+    implementation 'org.tensorflow:tensorflow-lite:2.9.0'
+    implementation 'org.tensorflow:tensorflow-lite-gpu:2.9.0'
```
Comment (Stan4402) [inline CameraXAdvanced/tflite/build.gradle:69]: 'org.tensorflow:tensorflow-lite-gpu:2.9.0'

File: CameraXAdvanced/gradle/wrapper/gradle-wrapper.properties [hunk 4]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.7.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-bin.zip
```
Comment (Stan4402) [inline CameraXAdvanced/gradle/wrapper/gradle-wrapper.properties:6]: distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-bin.zip

File: CameraXBasic/build.gradle (hunk 1 above)
Comment (Stan4402) [inline CameraXBasic/build.gradle:29]: 'com.android.tools.build:gradle:7.2.2'

File: CameraXAdvanced/gradle/wrapper/gradle-wrapper.properties (hunk 4 above)
Comment (Stan4402) [reply CameraXAdvanced/gradle/wrapper/gradle-wrapper.properties:6]: +6

Comment (Stan4402) [inline CameraXAdvanced/gradle/wrapper/gradle-wrapper.properties:6]: distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-bin.zip

**Review by Stan4402 — CHANGES_REQUESTED** [review]

**Pull request merged**